install_custom_module_name/after/Pods/Target Support Files/Pods-Example/Pods-Example.debug.xcconfig

```diff
@@ -2,7 +2,7 @@ FRAMEWORK_SEARCH_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/SocketRoc
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) COCOAPODS=1 kPTPusherClientLibraryVersion=@\"1.6.2\"
 LD_RUNPATH_SEARCH_PATHS = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
 OTHER_CFLAGS = $(inherited) -iquote "${PODS_CONFIGURATION_BUILD_DIR}/SocketRocket/SocketRocket.framework/Headers" -iquote "${PODS_CONFIGURATION_BUILD_DIR}/libPusher/Pusher.framework/Headers"
-OTHER_LDFLAGS = $(inherited) -framework "Pusher" -framework "SocketRocket"
+OTHER_LDFLAGS = $(inherited) -l"icucore" -framework "CFNetwork" -framework "Pusher" -framework "Security" -framework "SocketRocket"
```

Comment:
so now aggregates inherit all the system frameworks from all pods that are part of it.

Reply:
talked offline, this is true.